python/ray/worker.py

```diff
@@ -1040,6 +1040,17 @@ def array_custom_deserializer(serialized_obj):
         custom_serializer=array_custom_serializer,
         custom_deserializer=array_custom_deserializer)
 
+    def ordereddict_custom_serializer(obj):
+        return list(obj.keys()), list(obj.values())
```

Comment:
why not 1 list of tuples? ie, `return list(obj.items())`

this way the deserializer just needs to do `return collections.OrderedDict(obj)`

Reply:
Yeah, that's a good point! I'm currently trying to get rid of the temporary copy altogether, let's see how that goes :)